### PR TITLE
Add explanations to set the prod or sandbox environment

### LINKED ISSUE #28: Add explanations for RestClient's environment's setter functions
# Abstract

Need explanations in the PHP lib doc to set the environment with the new functions.

# Details

New functions the developers will use have been added, so it needs to be added to the doc.

# Requester

Julien Malherbe

### PR DESCRIPTION
**Pull request by PaulGirardin**
Fixes #28 

/!\ NEEDS NEW ZIP /!\

doc-restclient

doc-restclient2

### CODE REVIEW

File: source/includes/php/rest_client.md [hunk 1]
```diff
@@ -7,10 +7,10 @@
 require_once 'path/to/lib/shoprunback-php/init.php';
 
 $restClient = \Shoprunback\RestClient::getClient();
-$restClient->setApiBaseUrl('https://dashboard.shoprunback.com');
+$restClient->setApiBaseUrl('http://localhost:3000');
```
Comment (tildedash) [inline source/includes/php/rest_client.md]: no, you are the only developer who may use localhost as API base URL, others will use the sandbox/production.

File: source/includes/php/rest_client.md [hunk 2]
```diff
@@ -23,8 +23,47 @@ It is a **Singleton**, so you need to use `getClient` to use it.
...
+## Set parameters
+
+> Set RestClient parameters
+
+```php
+<?php
+require_once 'path/to/lib/shoprunback-php/init.php';
+
+$restClient = \Shoprunback\RestClient::getClient();
+$restClient->setApiBaseUrl('http://localhost:3000');
```
Comment (tildedash) [inline source/includes/php/rest_client.md]: use the new methods

File: source/includes/php/rest_client.md [hunk 3]
```diff
@@ -23,8 +23,42 @@ It is a **Singleton**, so you need to use `getClient` to use it.
...
+```
+
+> Use the sandbox environment
+
+```php
+<?php
+require_once 'path/to/lib/shoprunback-php/init.php';
+
+$restClient = \Shoprunback\RestClient::getClient();
+$restClient->useSandboxEnvironment();
```
Comment (tildedash) [inline source/includes/php/rest_client.md]: could be done in the same code snippet

Comment (tildedash) [reply source/includes/php/rest_client.md]: @PaulGirardin

File: source/includes/php/rest_client.md [hunk 4]
```diff
@@ -23,8 +23,37 @@ It is a **Singleton**, so you need to use `getClient` to use it.
...
+require_once 'path/to/lib/shoprunback-php/init.php';
+
+$restClient = \Shoprunback\RestClient::getClient();
+
+if (/* Are we on production environment? */) {
+  $restClient->useProductionEnvironment();
+} else {
+  $restClient->useSandboxEnvironment();
+}
+```
```
Comment (leo-martin) [inline source/includes/php/rest_client.md]: pas besoin d'un if else je pense
just montrer les 2 methodes de set d'environement

File: source/includes/php/rest_client.md [hunk 5]
```diff
@@ -23,8 +23,37 @@ It is a **Singleton**, so you need to use `getClient` to use it.
...
+
+If you want to **use the production or sandbox environment**, you can use `useProductionEnvironment` and `useSandboxEnvironment`.
+
+It will set **https://dashboard.shoprunback.com** for the production environment and **https://sandbox.dashboard.shoprunback.com** for the sandbox environment.
```
Comment (leo-martin) [inline source/includes/php/rest_client.md]: essaye de refacto ces 2 phrases la, t'utilises le mot production et sandbox 4 fois en 3 lignes

**Pull request closed without merging**